### ISSUE TITLE
source RPMs for rebuilding RPMs

### ISSUE THREAD

**Issue by JayDoubleu** [brave/brave-browser#13353]
Does brave have a SRPM files available to rebuild RPMs ?

**Comment by kjozwiak** [comment]
CCing @fmarier @mihaiplesa

**Comment by JayDoubleu** [comment]
Well, I am mainly looking at it from perspective of rpm-flatpaks which are essentially flatpaks built from source RPM packages available in fedora.

@fmarier is there an ETA or any timeline for source RPM itself being on the roadmap?
Another question which I guess is not exactly related to this but is brave truly a FOSS ? If so, having source RPMs would allow brave to be available natively  in Fedora and other RPM based distros.

**Comment by fmarier** [comment]
I just built the packages (`npm run create_dist Release`) and I don't see any source RPMs generated in `src/out/Release/`. This is the Chromium code that builds the RPMs: https://source.chromium.org/chromium/chromium/src/+/master:chrome/installer/linux/rpm/build.sh;l=84-120;drc=2fc4cadb85c37f3d834227b8ee8e35d7f6a937c0

**Comment by fmarier** [comment]
> Another question which I guess is not exactly related to this but is brave truly a FOSS ?

Yes, we release our code under MPL-2 and we don't include any proprietary code in Brave binaries. Things like Widevine DRM for example are downloaded and loaded at runtime (if not disabled in the settings).

**Comment by JayDoubleu** [comment]
So this is how chromium is being built in Fedora:
The chromium source is using this spec file template:
https://source.chromium.org/chromium/chromium/src/+/master:chrome/installer/linux/rpm/chrome.spec.template 

While Fedora RPM is being built from this:
https://src.fedoraproject.org/rpms/chromium/blob/master/f/chromium.spec 

So I guess I am essentially looking at a way to build brave without using NPM entirely .. 🤔

**Comment by bsclifton** [comment]
This issue hasn't had an update in a while - it's met the criteria for being stale. The issue is going to be closed for now. If this is still a problem, please leave a comment or re-open the issue. Thanks!
